scripts/setup-ubuntu.sh

```diff
@@ -244,6 +244,9 @@ PACKAGES+=" bash-static"
 # Needed by apt.
 PACKAGES+=" triehash"
 
+# Needed by sequoia-sq (dependency of apt)
+PACKAGES+=" nettle-dev"
```

Comment:
This adds 3MiB's to the docker image but it is infinitely preferable to host-building `nettle` as well.